<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CreateCapital</class>
 <widget class="QDialog" name="CreateCapital">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>352</width>
    <height>300</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>352</width>
    <height>300</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>352</width>
    <height>300</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>Segoe UI</family>
    <pointsize>11</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Recebimento de Capital</string>
  </property>
  <property name="windowIcon">
   <iconset resource="icons.qrc">
    <normaloff>:/Downloads/main_icon.png</normaloff>:/Downloads/main_icon.png</iconset>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog{
	background-image: url(:/Downloads/background2.jpg);
}
QLabel{
	color:#FF692E;
	font: 12pt &quot;Segoe UI&quot;;
}
QPushButton{
	background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                stop: 0 #ff9b58, stop: 0.4 #ff7037,
                                stop: 0.5 #ff7037, stop: 1.0 #ff9b58);
	border:none;
	padding: 10px;
	color:#FFF;
	margin-top: 10px;
}
QLineEdit{
	border:none;
	background-color:  transparent;
	padding:5px;
	border-bottom: 1px solid #000;
}
QDateEdit{
	padding:5px;
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="10" column="0">
      <widget class="QLabel" name="labelDesc">
       <property name="text">
        <string>Descrição:</string>
       </property>
      </widget>
     </item>
     <item row="9" column="0" alignment="Qt::AlignLeft">
      <widget class="QLineEdit" name="valorInput">
       <property name="placeholderText">
        <string>200.00</string>
       </property>
      </widget>
     </item>
     <item row="11" column="0" colspan="2">
      <widget class="QTextEdit" name="descInput"/>
     </item>
     <item row="12" column="0" alignment="Qt::AlignRight">
      <widget class="QPushButton" name="saveButton">
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="text">
        <string>Salvar</string>
       </property>
      </widget>
     </item>
     <item row="12" column="1" alignment="Qt::AlignLeft">
      <widget class="QPushButton" name="cancelButton">
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Cancelar</string>
       </property>
      </widget>
     </item>
     <item row="7" column="0" alignment="Qt::AlignLeft">
      <widget class="QLabel" name="labelValor">
       <property name="text">
        <string>Valor:</string>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QDateEdit" name="dateInput">
       <property name="cursor">
        <cursorShape>ArrowCursor</cursorShape>
       </property>
       <property name="calendarPopup">
        <bool>true</bool>
       </property>
       <property name="date">
        <date>
         <year>2000</year>
         <month>1</month>
         <day>1</day>
        </date>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="labelData">
       <property name="text">
        <string>Data:</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections/>
</ui>
